prepare-docs.rb

```diff
@@ -5,38 +5,100 @@
 require "net/http"
 require "uri"
 require "rexml/document"
-# require "nokogiri"
```

Comment:
What was that for and why did you remove it?

Reply:
Again a personal taste of mine: no dead code in productive things; but rather create a backlog item/issue ... as it was not used I removed it to clean things up a bit. It looks like it is a helper to process HTML & XML files: https://rubygems.org/gems/nokogiri/

I propose to remove it, if it is not used - but I'm also fine to keep it here (of course I would also like to know what the benefit would be).

Reply:
According to Git blame, `nogokiri` has been replaced by REXML back in 2017, so I'm fine with removing this line.